app/assets/stylesheets/application.css.scss

```diff
@@ -15,7 +15,11 @@
  *= require font-awesome
  */
 
- .social-links a {
+:root {
+  -phone-bg-color: #808080;
```

Comment:
Color literals like `#808080` should only be used in variable declarations; they should be referred to via variable everywhere else.